pandas/core/frame.py

```diff
@@ -4673,6 +4673,48 @@ def drop_duplicates(
         See Also
         --------
         DataFrame.value_counts: Count unique combinations of columns.
+
+        Examples
+        --------
+        Consider dataset containing ramen rating.
+
+        >>> brand = ['Yum Yum', 'Yum Yum', 'Indomie', 'Indomie', 'Indomie']
+        >>> df = pd.DataFrame({
+        ... 'brand': brand,
+        ... 'style': ['cup', 'cup', 'cup', 'pack', 'pack'],
+        ... 'rating': [4, 4, 3.5, 15, 5]
```

Comment:
```suggestion
        ...     'brand': brand,
        ...     'style': ['cup', 'cup', 'cup', 'pack', 'pack'],
        ...     'rating': [4, 4, 3.5, 15, 5]
```

Reply:
And if the `brand` line fits inline, that would be great, better than defining it in a variable.